rest/rest-api/src/main/java/org/apache/brooklyn/rest/domain/ConfigSummary.java

```diff
@@ -114,7 +116,8 @@ public ConfigSummary(ConfigKey<?> config, String label, Double priority, Boolean
                 : ImmutableList.<String>of();
         if (config.getType().isEnum()) {
             this.type = Enum.class.getName();
-            this.defaultValue = (config.getDefaultValue() == null) ? null : ((Enum<?>) config.getDefaultValue()).name();
+            this.defaultValue = config.getDefaultValue() instanceof Enum? ((Enum<?>) config.getDefaultValue()).name() : 
```

Comment:
`config.getDefaultValue()` can be null. This could then throw a NPE

Reply:
if it's null the check returns false and it calls to `convertToJsonPrimitive` which accepts null.